packages/flutter_tools/test/general.shard/web/devfs_web_ddc_modules_test.dart

```diff
@@ -116,1386 +61,1212 @@ void main() {
   setUp(() {
     httpServer = FakeHttpServer();
     linux = FakePlatform(environment: <String, String>{});
-    windows = FakePlatform(
-        operatingSystem: 'windows', environment: <String, String>{});
+    windows = FakePlatform(operatingSystem: 'windows', environment: <String, String>{});
```

Comment:
This exceeds 80 columns, so let's break it into multiple lines.
```suggestion
    windows = FakePlatform(
      operatingSystem: 'windows',
      environment: <String, String>{},
    );
```